src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -422,15 +420,15 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * With given feedbackConflictId, finds the conflict and sets it as solved.
+     * POST exercise/:exerciseId/feedbackConflict/:feedbackConflictId/solve-feedback-conflict : With given feedbackConflictId, finds the conflict and sets it as solved.
```

Comment:
This probably wants to be just `solve` now too?
```suggestion
     * POST exercise/:exerciseId/feedbackConflict/:feedbackConflictId/solve : With given feedbackConflictId, finds the conflict and sets it as solved.
```